core/DataAccess/ArchiveSelector.php

```diff
@@ -166,6 +166,9 @@ public static function getArchiveIds($siteIds, $periods, $segment, $plugins)
         $monthToPeriods = array();
         foreach ($periods as $period) {
             /** @var Period $period */
+            if ($period->getDateStart()->isLater(Date::now()->addDay(2))) {
+                continue; // avoid creating any archive tables in the future
+            }
```

Comment:
Should we add this code to ArchiveTableCreator? I can't think of a reason for code to create a table in the future (except possibly tests, though probably not). If some other code uses ArchiveTableCreator directly, this safeguard would be applied there too.

Reply:
Not sure if it might make sense for some custom plugins to create archives in the future. e.g. for some kind of prediction reports of whatever. Nevertheless the ArchiveSelector shouldn't trigger the creation

Reply:
Makes sense.